### ISSUE TITLE
Control-api gives warning when trying to connect redis and when greymatter calls are made

### ISSUE THREAD

**Issue by rahug20** [greymatter-io/helm-charts#819]
Warning:

`{"level":"warn","service":"gm-control-api","api version":"1.5.0","persister":"Redis","client.Ping":"DENIED Redis is running in protected mode because protected mode is enabled, no bind address was specified, no authentication password is requested to clients. In this mode connections are only accepted from the loopback interface. If you want to connect from external computers to Redis you may adopt one of the following solutions: 1) Just disable protected mode sending the command 'CONFIG SET protected-mode no' from the loopback interface by connecting to Redis from the same host the server is running, however MAKE SURE Redis is not publicly accessible from internet if you do so. Use CONFIG REWRITE to make this change permanent. 2) Alternatively you can just disable the protected mode by editing the Redis configuration file, and setting the protected mode option to 'no', and then restarting the server. 3) If you started the server manually just for testing, restart it with the '--protected-mode no' option. 4) Setup a bind address or an authentication password. NOTE: You only need to do one of the above things in order for the server to start accepting connections from the outside.","count":22,"time":"2020-11-24T20:38:22.696726565Z","message":"Unable to connect to Redis"}`

Workarund:

Running redis with `--protected-mode no` option

**Comment by chrisbsmith** [comment]
@justincely @tilleryd @bcmendoza - Can someone take a look at this and chime in with any thoughts?  We shouldn't be running a redis instance in protected mode and we should ensure that control-api talks to redis with some authentication mechanisms in place.

**Comment by bcmendoza** [comment]
@rahug20 I'm not seeing this issue in the current Helm charts configuration. Our Helm charts don't bundle a Redis config file. Are you using an external Redis provider that has protected mode enabled?

**Comment by rahug20** [comment]
> @rahug20 I'm not seeing this issue in the current Helm charts configuration. Our Helm charts don't bundle a Redis config file. Are you using an external Redis provider that has protected mode enabled?

Yes! I had to use that that because the the redis image we have was showing issues.

**Comment by bcmendoza** [comment]
@rahug20 Ah okay. The issue related to setting `protected-mode` has to do with Redis configuration, and not our helm-charts. What is the issue with the Redis image referenced in our configs?
